cylc/flow/network/schema.py

```diff
@@ -1219,6 +1219,23 @@ def description(self):
         return ''
 
 
+class BroadcastSetting(GenericScalar):
+    """A [runtime] key=value configuration for a namespace.
+
+    Should be a key=value pair where sections are wrapped with square
+    brackets.
+
+    Examples:
+        script=true
+        [environment]ANSWER=42
```

Comment:
These examples are true of the user input (both in the UI and on the CLI), however, the transfer format and GraphiQL differ which makes this rather awkward to document.